### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by Bajix**
Update to use latest core & cleanup

### CODE REVIEW

**Comment by AZholtkevych** [comment]
@KronicDeth could you please help to review this PR. Thank you so much

**Comment by AZholtkevych** [comment]
FYI @nelson-glauber

File: jni_bindings/Cargo.toml [hunk 1]
```diff
@@ -10,9 +10,9 @@ name = "liveview_native_core"
...
-jni = "0.21.0"
-log = "0.4.17"
+cranelift-entity = "0.96"
+jni = "0.21"
+log = "0.4"
 log-panics = { version = "2", features = ["with-backtrace"] }
-android_logger = "0.13.1"
-liveview-native-core = { git = "https://github.com/liveview-native/liveview-native-core.git" }
+android_logger = "0.13.3"
+liveview-native-core = { git = "https://github.com/liveview-native/liveview-native-core.git", rev = "fadf150a987c9fd07baefa9ed6b19635ed93f1e5" }
```
Comment (KronicDeth) [inline jni_bindings/Cargo.toml]: This is on `main` with [https://github.com/liveview-native/liveview-native-core/commit/fadf150a987c9fd07baefa9ed6b19635ed93f1e5 merged in](https://github.com/liveview-native/liveview-native-core/pull/32) merged.

**Review by KronicDeth — APPROVED** [review]

**Comment by AZholtkevych** [comment]
@Bajix this PR has been approved. I think we can merge that. Thanks

**Pull request merged**